OpenRA.Mods.Common/Traits/Voiced.cs

```diff
@@ -37,6 +39,7 @@ public Voiced(Actor self, VoicedInfo info)
 		}
 
 		public string VoiceSet { get { return Info.VoiceSet; } }
+		public string SelectVoice { get { return Info.SelectVoice; } }
```

Comment:
Nah. We just split the voices off. I'd personally rather keep the voices in `Voiced`, even this one.